src/mlang/backend_compilers/bir_to_dgfip_c.ml

```diff
@@ -284,8 +284,8 @@ let rec generate_c_expr (e : expression Pos.marked)
         Dand
           ( Dand
               ( Dvar (Local idx_var, Def),
-                Dbinop
-                  ("<=", Dvar (Local idx_var, Val), Dlit (float_of_int size)) ),
+                Dbinop ("<", Dvar (Local idx_var, Val), Dlit (float_of_int size))
+              ),
             Daccess (Pos.unmark var, Def, idx_var) )
```

Comment:
This (and the Java equivalent) has technically no link with OCaml backend and should be cherry-picked in its own PR

Reply:
Done. Add dependency to PR#196 to this one (administrative dependency, so to speak)